airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1454,6 +1454,9 @@ def execute(self, context: Context):
             if self.deferrable:
                 self.defer(
                     trigger=DataprocSubmitTrigger(
+                        dag_id=self.dag_id,
+                        task_id=self.task_id,
+                        run_id=context.get("run_id"),
```

Comment:
When will we get a `None` run_id?

Reply:
removed it completely

Reply:
Is this missed? It seems to still be here

Reply:
removed it